google/cloud/pubsub/internal/publisher_metadata_test.cc

```diff
@@ -38,6 +42,22 @@ class PublisherMetadataTest : public ::testing::Test {
         context, method, google::cloud::internal::ApiClientHeader());
   }
 
+  void ValidateNoUserProject(grpc::ClientContext& context) {
+    auto md = validate_metadata_fixture_.GetMetadata(context);
+    EXPECT_THAT(md, Not(Contains(Pair("x-goog-user-project", _))));
+  }
+
+  void ValidateTestUserProject(grpc::ClientContext& context) {
+    auto md = validate_metadata_fixture_.GetMetadata(context);
+    EXPECT_THAT(md, Contains(Pair("x-goog-user-project", "test-project")));
+  }
+
+  static Options TestOptions(std::string const& user_project) {
+    return user_project.empty()
+               ? Options{}
+               : Options{}.set<UserProjectOption>("test-project");
```

Comment:
Did you mean to say `set<UserProjectOption>(user_project)` in these?

Reply:
Yes, thanks for catching that.